run/GCHP/runScriptSamples/gchp.batch_job.sh

```diff
@@ -0,0 +1,157 @@
+#
+#PBS -S /bin/bash
+#PBS -l select=2:ncpus=24:mpiprocs=24:model=bro
+#PBS -l walltime=02:00:00
+#PBS -j oe
+#PBS -W group_list=xXXXX
+#PBS -m e
+#
+# DESCRIPTION      Sample GCHP batch job script for Pleiades
+# SCHEDULER        PBS
+# DATE             2021-02-18
+# SIMULATION       C90, 2019-01-01 to 2019-01-02
+# RESOURCES        2 nodes (Broadwell; model=bro), 24 processes per node
+# SEE ALSO         qsub man pages, http://gchp.rtfd.io/
+#
+# -----------------
+# Example 2: LSF
+# 
+#BSUB -q rvmartin
+#BSUB -n 72
+#BSUB -W 2:00
+#BSUB -R "rusage[mem=120GB] span[ptile=36] select[mem < 2TB]"
+#BSUB -a 'docker(geoschem/gchp:13.0.0-beta.1-13-g924e47f)
+#BSUB -o lsf-%J.txt
+#
+# DESCRIPTION      Sample GCHP batch job script for Compute1 (Docker-based environments)
+# SCHEDULER        LSF
+# DATE             2021-02-19
+# SIMULATION       C90, 2019-01-01 to 2019-01-02
+# RESOURCES        2 nodes, 36 processes per node
+# SEE ALSO         bsub man pages, http://gchp.rtfd.io/
+#
+# -----------------
+# Example 3: SLURM
+#
+#SBATCH -n 60
+#SBATCH -N 2
+#SBATCH -t 2:00:00
+#SBATCH -p huce_intel
+#SBATCH --mem=110G
+#SBATCH --mail-type=ALL
+#
+# DESCRIPTION      Sample GCHP batch job script for Cannon
+# SCHEDULER        SLURM
+# DATE             2021-02-19
+# SIMULATION       C90, 2019-01-01 to 2019-01-02
+# RESOURCES        2 nodes, 30 processes per node
+# SEE ALSO         sbatch man pages, http://gchp.rtfd.io/
+#
+
+#################################################################
+#
+# LOAD LIBRARIES - 2 options
+#
+# Option 1: Define environment settings in external file, link to it from
+#           run directory symlink gchp.env, and source it here.
+# 
+#    source gchp.env
+#
+# Option 2: Define environment settings directly in this file. Two
+#           examples are below.
+# 
+#    Example 1: PBS on NASA Pleiades
+#
+#       module purge
+#       module use /nobackup/lbindle/modulefiles
+#       module load gchp-env/2021.06-gnu
+#
+#    Example 2: LSF on WashU Compute 1 (uses docker containers)
+#
+#       . /spack/share/spack/setup-env.sh
+#       module load openmpi-4.0.1-gcc-9-sdj47y5
+#
+
+#################################################################
+#
+# ADDITIONAL PRE-RUN CONFIGURATION
+#
+# If a subsequent command fails, treat it as fatal (don't continue)
+set -e
+
+# For remainder of script, echo commands to the job's log file
+set -x
+
+# Unlimit resources to prevent OS killing GCHP due to resource usage/
+# Alternatively you can put this in your environment file.
+ulimit -c 0                  # coredumpsize
+ulimit -l unlimited          # memorylocked
+ulimit -u 50000              # maxproc
+ulimit -v unlimited          # vmemoryuse
+ulimit -s unlimited          # stacksize
+
+#################################################################
+#
+# PRE-RUN COMMANDS
+#
+# Print loaded modules
+module list     
+
+# Define log name to include simulation start date
+start_str=$(echo $(cat cap_restart) | sed 's/ /_/g')
```

Comment:
@lizziel I think this (and similar occurrences) is a little neater written as:

```
sed 's/ /_/g' cap_restart
```

Reply:
Indeed. Thanks for the tip! I'll push that update.